oauth2client/contrib/gce.py

```diff
@@ -134,8 +133,8 @@ def _refresh(self, http_request):
             self._retrieve_info(http_request)
             self.access_token, self.token_expiry = _metadata.get_token(
                 http_request, service_account=self.service_account_email)
-        except httplib2.HttpLib2Error as e:
-            raise client.HttpAccessTokenRefreshError(str(e))
+        except transport.transport_error() as err:
```

Comment:
No, I'm just saying `httplib2.HttpLib2Error` doesn't need to be the error that communicates the failure. Something more generic would suffice.

Though now that you mention it, the only users of `_metadata.get` is `gce.AppAssertionCredentials._refresh` (via `_metadata.get_token`) and `gce.AppAssertionCredentials._retrieve_info` (via `_metadata.get_service_account_info`) so it'd be easy to change. As currently written,  `get_service_account_info` would return `None` and `get_token` would result in a `TypeError` (via `__getitem__` on `None`).